src/compounds/ItemPage/ItemPage.jsx

```diff
@@ -1,21 +1,26 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { Button, Row, Col } from 'react-bootstrap'
 import Image from '../../components/Image/Image'
-import TextBox from '../../components/TextBox/TextBox'
 import Title from '../../components/Title/Title'
-import './item-page.css'
 
 const ItemPage = ({ title, titleStyle, description, descriptionStyle, img }) => {
   // sets a default width while still allowing width to be overridden
   img = { width: 400, ...img }
 
   return (
-    <main className='container item-page'>
-      <Title title={title} style={titleStyle} />
-      <div className='item-page-details'>
-        <TextBox text={description} style={descriptionStyle} size='medium' />
-        <Image {...img} />
-      </div>
+    <main className='container py-5'>
+      <Title title={title} style={titleStyle} addClass='text-center text-md-start'/>
```

Comment:
can we change it so that the text and button on the smaller responsive versions of this page are still left aligned?

Reply:
yes but it looks kind of odd if the image is centered
<img width="678" alt="image" src="https://user-images.githubusercontent.com/73361970/211827902-696e688e-1154-451a-855c-116c7329e0bd.png">

Reply:
yes please.

Reply:
@alishaevn ok, requested changes made